### ISSUE TITLE
steam lookup command

### ISSUE THREAD

**Issue by naterchrdsn** [Spudnik-Group/Spudnik#78]
```
Usage
.steam [custom_url_name] | .steam game [game name]

Example
<f0rked> .steam f0rked
<UB3R-B0T> SVARRENGLOSSEN XIII: Last Online: 16 days ago | Hours Played [last two weeks]: 0 | Steam
           ID: STEAM:0:1:3266323
<EvilMaverick> .steam tf2 atreus
<UB3R-B0T> Playtime: 70.9 hrs past 2 weeks | Total Points: 134,479 | Most Points: 3,592 (as Medic) | 
           Most Kills: 57 (as Heavy) | Achievements: 202 of 245 (82%) | Fav Class: Medic @ 494.8 hours
<arch> .steam game rocket league
<UB3R-B0T> http://store.steampowered.com/app/252950/
```

https://www.npmjs.com/package/steamapi

**Comment by naterchrdsn** [comment]
https://owo.whats-th.is/95de18.png

**Comment by develogix** [comment]
closed by #327
